app/assets/src/components/common/DetailsSidebar/PipelineStepDetailsMode/PipelineStepDetailsMode.jsx

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import cs from "./pipeline_step_details_mode.scss";
+
+class PipelineStepDetailsMode extends React.Component {
+  renderInputFiles() {
+    const { inputFiles } = this.props;
+    if (inputFiles && inputFiles.length) {
```

Comment:
Maybe just `return null` otherwise.... Better for readability because it avoids having to follow all the way down to see what happens.